### ISSUE TITLE
record.errors is always non-empty because it contains a "null" error

### ISSUE THREAD

**Issue by ericcj** [NetSweet/netsuite#590]
at least after an upsert maybe after any request it looks like this "errors":[{"type":null,"code":null,"message":null}]

**Comment by gbs4ever** [comment]
yes i just had this last week this code used to work  ```@transfer_order.errors.nil?  vs   @transfer_order.errors.all? { |error| error.message.nil? }  ```but now
